fuzzers/046-clk-bufg-muxed-pips/generate.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+from prjxray.segmaker import Segmaker
+import os
+import os.path
+import re
+
+
+def bitfilter(frame, word):
+    if frame < 26:
```

Comment:
`return frame >= 26`

Reply:
This is of course a valid suggestion, but in my opinion the explicit returns make it easier to understand what is filtered out by the bitfilter.